test/e2e/specs/onboarding/onboarding__site-assembler.ts

```diff
@@ -55,14 +54,11 @@ describe( 'Onboarding: Site Assembler', () => {
 		} );
 
 		it( 'Enter Onboarding flow for the selected domain', async function () {
-			await page.waitForURL(
-				DataHelper.getCalypsoURL(
-					`/setup/site-setup/goals?siteSlug=${ selectedFreeDomain }&siteId=${ newSiteDetails.blog_details.blogid }`
-				),
-				{
-					timeout: 30 * 1000,
-				}
-			);
+			await page.waitForURL( /setup\/site-setup\/goals/, {
+				timeout: 20 * 1000,
```

Comment:
I felt the original assertion is unnecessarily restrictive, and once we allow multi-domain selection the mental model of "one site, one domain" during onboarding breaks down.